bci_tester/data.py

```diff
@@ -325,9 +327,11 @@ def create_BCI(
     return pytest.param(
         DerivedContainer(
             base=baseurl,
-            containerfile=""
-            if bci_type == ImageType.OS_LTSS
-            else _BCI_REPLACE_REPO_CONTAINERFILE,
+            containerfile=(
```

Comment:
is this a automatic code formatter change?

Reply:
yes, I forced it to be more readable by adding the braces